python/cudf/cudf/_lib/strings/findall.pyx

```diff
@@ -8,6 +9,7 @@ from cudf._lib.column cimport Column
 from cudf._lib.cpp.column.column cimport column
 from cudf._lib.cpp.column.column_view cimport column_view
 from cudf._lib.cpp.scalar.scalar cimport string_scalar
+from cudf._lib.cpp.strings.contains cimport regex_flags
```

Comment:
We usually prefer a mapping from one `hpp` to one `pxd`, which is being broken by `flags.hpp` definitions in `contains.pxd`. Now that it's being used in multiple places, could we move the definition of `ctypedef enum regex_flags` to a new file `flags.pxd` and update the imports in `contains.pxd`, `contains.pyx`, etc.? It's okay with me if you want to do this in a follow-up PR.

Reply:
Ok. That makes sense. I did the same thing in this PR: https://github.com/rapidsai/cudf/pull/10192/files#diff-b16e9a967b36a1aa785b47dd6b66179ce0f6010f5fcb9079cd6e04067f32afe5R8
I'll look at fixing it here.

Reply:
Since this PR is done. I'm going to merge this PR and make the above change in #10208 where this is needed as well. This way I can fix it in both places without a third PR.

Reply:
This has been now completed in #10192